PostgresQLScript/seedAvatars.js

```diff
@@ -38,7 +38,7 @@ const insertAvatars = (start) => {
 
 const createAvatars = () => {
   dbt.none(`CREATE TABLE avatars(id INTEGER PRIMARY KEY, url1 TEXT, url2 TEXT, 
-    url3 TEXT, url4 TEXT, url5 TEXT, REFERENCE users);`)
+    url3 TEXT, url4 TEXT, url5 TEXT, avatar_id INTEGER REFERENCES users);`)
```

Comment:
@Sr5x4FC3s this may be a lack of knowledge on SQL syntax on my part -- but how come this says `REFERENCES` here vs. `REFERENCE` above without the "s"? are those different commands?

Reply:
TBH i don't know the exact answer to that question, I changed it because it was throwing me errors in the terminal when i tried to run the file; guessing because it didn't like the syntax.